modules/installation-vsphere-machines.adoc

```diff
@@ -0,0 +1,142 @@
+----
+
+. Obtain the {op-system} OVA image:
++
+----
+$ curl -J -L -O https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190412.1/rhcos-410.8.20190412.1-vmware.ova
+----
+
+. In the vSphere Client, create a folder in your datacenter to store your VMs.
+.. Click the *VMs and Templates* view.
+.. Right-click the name of your datacenter.
+.. Click *New Folder* -> *New VM and Template Folder*.
+.. In the window that is displayed, enter the folder name. The folder name must
+match the cluster name that you specified in the `install-config.yaml` file.
+
+. In the vSphere Client, create a template for the OVA image.
++
+[NOTE]
+====
+In the following steps, you use the same template for all of your cluster
+machines and provide the location for the Ignition config file for that machine
+type when you provision the VMs.
+====
+.. From the *Hosts and Clusters* tab, right-click your cluster's name and
+click *Deploy OVF Template*.
+.. On the *Select an OVF* tab, specify the name of the {op-system} OVA file
+that you downloaded.
+.. On the *Select a name and folder* tab, set a *Virtual machine name*, such
+as {op-system}, and click the name of the your vSphere cluster.
+.. On the *Select a compute resource* tab, click the name of the your vSphere
+cluster.
+.. On the *Select storage* tab, configure the storage options for your VM.
+*** Select *Thin Provision*.
+*** Select the datastore that you specified in your `install-config.yaml` file.
+.. On the *Select network* tab, specify the the network that you configured
+for the cluster, if available.
+.. If you plan to use the same template for all cluster machine types, do not
+specify values on the *Customize template* tab.
+
+. After the template deploys, deploy a VM for a machine in the cluster.
+.. Right-click the template's name and click *Clone* -> *Clone to Virtual Machine*.
+.. On the *Select a name and folder* tab, specify a name for the VM. You might
+include the machine type in the name, such as `control-plane-0` or `compute-1`.
+.. On the *Select a name and folder* tab, select the name of the folder that
+you created for the cluster.
+.. On the *Select a compute resource* tab, select the name of a host in your
+datacenter.
+.. Optionally, on the *Select storage* tab, customize the storage options.
+.. On the *Select clone options*, select
+*Customize this virtual machine's hardware*.
+.. On the *Customize hardware* tab, click *VM Options* -> *Advanced*.
+*** From the *Latency Sensitivity* list, select *High*.
+*** Click *Edit Configuration*, and on the *Configuration Parameters* window,
+click *Add Configuration Params*. Define the following parameter names and values:
+**** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded
+Ignition config file for this machine type.
+**** `guestinfo.ignition.config.data.encoding`: Specify `base64`.
```

Comment:
As an alternate, the user can set these properties after doing the clone. On the VM, open the Configure tab. Select Settings/vApp Options from the left menu. In the Properties section, the two guestinfo properties will be listed. The user can select a property and set its value by pressing SET VALUE. This may be easier for the user than having to manually enter the name of the property.

Reply:
@staebler, yes. If users set these values after the VM is cloned, do they also need to set the disk_uuid?

Reply:
The `disk.EnableUUID` still needs to be set. That can be set either before or after the clone: Whichever the user prefers.